src/cpp/session/resources/tutorial_resources/index.html

```diff
@@ -7,9 +7,23 @@
    </head>
 
    <body class="rstudio-tutorials" style="visibility: hidden;">
+
       <div class="rstudio-tutorials-container">
```

Comment:
Instead of a `div` use `<main class="rstudio-tutorials-container">`. This is more semantically correct and provides more context on the page layout to a screen reader user.

If `main` causes styling issues, could instead add `role="main"` to the `div` and achieve the same result.